testsuite/features/step_definitions/common_steps.rb

```diff
@@ -586,26 +586,17 @@
   os_version, os_family = get_os_version($minion)
 
   # Distribution Pool and Update
-  if os_family =~ /^opensuse/
-    repos = "openSUSE-Leap-#{os_version}-Pool openSUSE-Leap-#{os_version}-Update"
-    puts $minion.run("zypper mr --enable #{repos}")
-  elsif os_version =~ /^15/
-    repos, _code = $minion.run('zypper lr | grep SLE-Module-Basesystem | cut -d"|" -f2')
-    puts $minion.run("zypper mr --enable #{repos.gsub(/\s/, ' ')}")
-  else
-    arch, _code = $minion.run('uname -m')
-    repos = "SLE-#{os_version}-#{arch.strip}-Pool SLE-#{os_version}-#{arch.strip}-Update"
-    puts $minion.run("zypper mr --enable #{repos}")
-  end
+  repos = "os_pool_repo os_update_repo"
+  puts $minion.run("zypper mr --enable #{repos}")
```

Comment:
I'd inline `repos` as it is only used once, overwritten later, and requires one to look at two lines instead of one while still being very simple.

Reply:
It's for homogeneity along all this code, between cases where determined dynamically and cases where hardcoded.

It does not cost much, it's a local variable (deleted as soon as execution flow hits `end`).